### PR TITLE
You have to wave your hands around to use the Force

### LINKED ISSUE #7754: Telekenisis Bug
(WEB REPORT BY: gunosaguki REMOTE: 158.69.55.2:7777)
Having no Arms prevents you from Using telekenisis

### PR DESCRIPTION
**Pull request by Intigracy**
fixes #7754 
tested-stamp

### CODE REVIEW

**Comment by gbasood** [comment]
You have to wave your arms around to use the Force

**Comment by clusterfack** [comment]
You have to compile to get your PR merged

**Comment by Intigracy** [comment]
yeah I know
I'm actually testing it and nothing I change it to to check for hands is working aaaaaaaaaaa

**Comment by ghost** [comment]
Has this been tested?

**Comment by Intigracy** [comment]
I don't lie about testing.

Read the OP nerd.

**Comment by gbasood** [comment]
Add a changelog
Also hopefully that'll trigger Travis

**Comment by Intigracy** [comment]
I'm triggered.

**Comment by Intigracy** [comment]
https://travis-ci.org/d3athrow/vgstation13/builds/115694344

**Comment by Exxion** [comment]
I'm proud of my bouncing baby meme

**Comment by gbasood** [comment]
``` DM
if(user.put_in_hands(O))
    user.put_in_active_hand(O)
```

w h y

**Comment by Intigracy** [comment]
The first is checking if you can do it
The second is doing it
They're two different procs.

**Comment by Exxion** [comment]
I'm pretty sure both of them are both checks _and_ actions

**Comment by Intigracy** [comment]
idc it only functions the way I want it to when I have both, so I'm leaving it that way.

**Comment by Intigracy** [comment]
fixed another issue that appeared in the issue report

File: code/_onclick/telekinesis.dm [hunk 1]
```diff
@@ -32,18 +32,24 @@ var/const/tk_maxrange = 15
 		return
 
 	var/obj/item/tk_grab/O = new(src)
-	user.put_in_active_hand(O)
-	O.host = user
-	O.focus_object(src)
+	if(user.put_in_hands(O))
+		user.put_in_active_hand(O)
```
Comment (clusterfack) [inline code/_onclick/telekinesis.dm]: This literally makes no sense, as proc/put_in_hands calls this proc

Comment (Intigracy) [inline code/_onclick/telekinesis.dm]: idk why it works.

The behavior with the current code acts like it picks up the item, and it glows like you're moving it with TK, but it's immediately dropped before the icon even shows up in hand on the player's end.

As-is the code works where it doesn't show you the shinies and gives you the message, you're free to help me find out why.

Comment (clusterfack) [inline code/_onclick/telekinesis.dm]: Is that the behavior with put in hands only or with put in active hand only?

Comment (Intigracy) [inline code/_onclick/telekinesis.dm]: I think it was put_in_active_hand only. I was cycling through all the available put in hand procs until I found one that returned false, so maybe I don't even need the put_in_active_hand. I'll check.

**Comment by Intigracy** [comment]
Tested, it works.

**Comment by Intigracy** [comment]
Sorry Exxion I had to change your meme a tad to be more understandable.

**Pull request merged**